<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/icons.qrc">
    <normaloff>:/icons/scibeats.png</normaloff>:/icons/scibeats.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Export"/>
    <addaction name="action_ExportSelection"/>
    <addaction name="action_Property"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Play">
    <property name="title">
     <string>&amp;Play</string>
    </property>
    <addaction name="action_Loop"/>
    <addaction name="separator"/>
    <addaction name="action_Play"/>
    <addaction name="action_PlayAll"/>
    <addaction name="action_Pause"/>
    <addaction name="action_Stop"/>
    <addaction name="action_GoToStart"/>
    <addaction name="action_GoToEnd"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_ZoomIn"/>
    <addaction name="action_ZoomOut"/>
    <addaction name="action_FitInWindow"/>
    <addaction name="action_ZoomToSelection"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_AboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Play"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Loop"/>
   <addaction name="separator"/>
   <addaction name="action_PlayAll"/>
   <addaction name="action_Play"/>
   <addaction name="action_Pause"/>
   <addaction name="action_Stop"/>
   <addaction name="action_GoToStart"/>
   <addaction name="action_GoToEnd"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Data source</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="action_Export">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-save-as.png</normaloff>:/icons/action-save-as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export...</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-exit.png</normaloff>:/icons/action-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit...</string>
   </property>
  </action>
  <action name="action_Loop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-playback-loop.png</normaloff>:/icons/action-playback-loop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Loop</string>
   </property>
  </action>
  <action name="action_Play">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-playback-start.png</normaloff>:/icons/action-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
  </action>
  <action name="action_PlayAll">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-playback-playall.png</normaloff>:/icons/action-playback-playall.png</iconset>
   </property>
   <property name="text">
    <string>Play &amp;All</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-playback-pause.png</normaloff>:/icons/action-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pa&amp;use</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-playback-stop.png</normaloff>:/icons/action-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="action_GoToStart">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-playback-backward.png</normaloff>:/icons/action-playback-backward.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Go to start</string>
   </property>
  </action>
  <action name="action_GoToEnd">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-playback-forward.png</normaloff>:/icons/action-playback-forward.png</iconset>
   </property>
   <property name="text">
    <string>Go to &amp;end</string>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
  </action>
  <action name="action_FitInWindow">
   <property name="text">
    <string>&amp;Fit in Window</string>
   </property>
  </action>
  <action name="action_ZoomToSelection">
   <property name="text">
    <string>Zoom to &amp;Selection</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/scibeats.png</normaloff>:/icons/scibeats.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_AboutQt">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-about-qt.png</normaloff>:/icons/action-about-qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="action_Property">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-properties.png</normaloff>:/icons/action-properties.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Property...</string>
   </property>
  </action>
  <action name="action_ExportSelection">
   <property name="icon">
    <iconset resource="../qrc/icons.qrc">
     <normaloff>:/icons/action-save-selection-as.png</normaloff>:/icons/action-save-selection-as.png</iconset>
   </property>
   <property name="text">
    <string>Export Selection...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
